src/components/Dialog/Dialog.tsx

```diff
@@ -51,49 +51,49 @@ export class Dialog extends React.Component<IDialogProps, any> {
     let groupings = this._groupChildren();
 
     if (subText) {
-      subTextContent = <p className='ms-Dialog-subText' id={ id + '-subText'}>{ subText }</p>;
+      subTextContent = <p className='ms-Dialog-subText' id={ id + '-subText' }>{ subText }</p>;
     }
 
     // @temp tuatology - Will adjust this to be a panel at certain breakpoints
     if (responsiveMode >= ResponsiveMode.small) {
       return (
         <Layer onLayerMounted={ onLayerMounted }>
           <Popup
-            className= { dialogClassName }
+            className={ dialogClassName }
             role='dialog'
             ariaLabelledBy={ title ? id + '-title' : '' }
             ariaDescribedBy={ subText ? id + '-subText' : '' }
             onDismiss={ onDismiss }
             >
-              <Overlay isDarkThemed={ isDarkOverlay } onClick={ isBlocking ? null : onDismiss } />
-              <FocusTrapZone
-                className={ css('ms-Dialog-main', this.props.containerClassName) }
-                elementToFocusOnDismiss={ elementToFocusOnDismiss }
-                isClickableOutsideFocusTrap={ !isBlocking }
-                ignoreExternalFocusing={ ignoreExternalFocusing }
-                forceFocusInsideTrap={ forceFocusInsideTrap }
-                firstFocusableSelector={ firstFocusableSelector }>
-                <div className='ms-Dialog-header'>
-                  <p className='ms-Dialog-title' id={ id + '-title'}>{ title }</p>
-                  <div className='ms-Dialog-topButton'>
-                    <Button
-                      className='ms-Dialog-button ms-Dialog-button--close'
-                      buttonType={ ButtonType.icon }
-                      icon='Cancel'
-                      rootProps={ { title: closeButtonAriaLabel } }
-                      ariaLabel={ closeButtonAriaLabel }
-                      onClick={ onDismiss }
+            <Overlay isDarkThemed={ isDarkOverlay } onClick={ isBlocking ? null : onDismiss } />
+            <FocusTrapZone
+              className={ css('ms-Dialog-main', this.props.containerClassName) }
+              elementToFocusOnDismiss={ elementToFocusOnDismiss }
+              isClickableOutsideFocusTrap={ this.props.isClickableOutsideFocusTrap ? this.props.isClickableOutsideFocusTrap : !isBlocking }
```

Comment:
This should just be isClickableOutsideFocusTrap ? isClickableOutsideFocusTrap: !isBlocking